vsintegration/src/FSharp.Editor/Common/CancellableTasks.fs

```diff
@@ -467,6 +467,10 @@ module CancellableTasks =
         // Low priority extensions
         type CancellableTaskBuilderBase with
 
+            [<NoEagerConstraintApplication>]            
+            member inline _.Source(awaiter: CancellableTask<unit array>) = 
+                (fun (token) -> (awaiter token :> Task).GetAwaiter())
```

Comment:
Is it needed now, since we now have `ignore`?

Reply:
Well, it saves us having to use ignore explicitly.